tf2onnx/tfonnx.py

```diff
@@ -766,6 +766,12 @@ def concat_op(ctx, node, name, args):
 def concatv2_op(ctx, node, name, args):
     # T output = ConcatV2(T values, Tidx axis, @int N, @type Tidx)
     # T concat_result = Concat(T inputs, @INT axis)
+    # if any input is empty, remove this concat op
+    # NOTE: workaround for https://github.com/Microsoft/onnxruntime/issues/681
+    for i, inp in enumerate(node.inputs):
+        if inp.is_const() and inp.get_tensor_value() == []:
```

Comment:
so original there are 2 inputs, remove one of them, concat having one input is valid, right? can you confirm on ONNX side and ONNXRuntime side?

Reply:
yes, both of ONNX and ONNXRuntime accept this kind of input.